tests/e2e/performance/test_pod_reattachtime.py

```diff
@@ -19,21 +19,21 @@ class TestPVCCreationPerformance(E2ETest):
     """
 
     @pytest.fixture()
-    def base_setup(self, request, interface_iterate, storageclass_factory):
+    def base_setup(self, request, interface_iterate):
         """
         A setup phase for the test
         Args:
             interface_iterate: A fixture to iterate over ceph interfaces
-            storageclass_factory: A fixture to create everything needed for a
-                storageclass
+
         """
         self.interface = interface_iterate
 
         if self.interface.lower() == "cephfs":
             self.interface = constants.CEPHFILESYSTEM
+            self.sc_obj = constants.DEFAULT_STORAGECLASS_CEPHFS
         if self.interface.lower() == "rbd":
             self.interface = constants.CEPHBLOCKPOOL
-        self.sc_obj = storageclass_factory(self.interface)
+            self.sc_obj = constants.DEFAULT_STORAGECLASS_RBD
```

Comment:
Any reason the test won't be run on cephFS?

Reply:
it will run on both (RBD and CephFS), it will not create new SC

Reply:
you can see it ran on RBD & CephFS in the Jenkins run results